server/models/userProfile.js

```diff
@@ -0,0 +1,62 @@
+const mongoose = require('mongoose')
+
+const userProfile = new mongoose.Schema({
+    firstName: {
+        type: 'string',
+        required: false
+    },
+    lastName: {
+        type: 'string',
+        required: false
+    },
+    gender: {
+        type: 'string',
+        required: false
+    },
+    birthDay: {
+        type: Date,
+        default: null
+    },
+    email: {
```

Comment:
Please remove the `email` field as the User model already contains a field for emails

Reply:
ok

Reply:
Please remove this field